adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegral.m

```diff
@@ -13,81 +13,95 @@
 // limitations under the License.
 
 #import "GADMediationAdapterMintegral.h"
-#import "GADMAdapterMintegralBannerAdLoader.h"
+#import "GADMediationAdapterMintegralConstants.h"
 #import "GADMAdapterMintegralExtras.h"
+#import "GADMAdapterMintegralRTBRewardedAdLoader.h"
+#import "GADMAdapterMintegralRTBBannerAdLoader.h"
+#import "GADMAdapterMintegralRTBInterstitialAdLoader.h"
+#import "GADMAdapterMintegralRTBNativeAdLoader.h"
+#import "GADMAdapterMintegralUtils.h"
+#import "GADMAdapterMintegralRewardedAdLoader.h"
+#import "GADMAdapterMintegralBannerAdLoader.h"
 #import "GADMAdapterMintegralInterstitialAdLoader.h"
 #import "GADMAdapterMintegralNativeAdLoader.h"
-#import "GADMAdapterMintegralRewardedAdLoader.h"
-#import "GADMAdapterMintegralUtils.h"
-#import "GADMediationAdapterMintegralConstants.h"
 
 #import <MTGSDK/MTGSDK.h>
 #import <MTGSDKBidding/MTGBiddingSDK.h>
 
+
 @implementation GADMediationAdapterMintegral {
-  /// Mintegral banner ad.
-  GADMAdapterMintegralBannerAdLoader *_bannerAd;
+    /// Mintegral RTB banner ad.
+    GADMAdapterMintegralRTBBannerAdLoader *_rtbBannerAd;
+
+    /// Mintegral RTB interstitial ad.
+    GADMAdapterMintegralRTBInterstitialAdLoader *_rtbInterstitialAd;
+
+    /// Mintegral RTB native ad.
+    GADMAdapterMintegralRTBNativeAdLoader *_rtbNativeAd;
 
-  /// Mintegral interstitial ad.
-  GADMAdapterMintegralInterstitialAdLoader *_interstitialAd;
+    /// Mintegral RTB rewarded ad.
+    GADMAdapterMintegralRTBRewardedAdLoader * _rtbRewardedAd;
+    
+    /// Mintegral waterfall banner ad.
+    GADMAdapterMintegralBannerAdLoader *_bannerAd;
```

Comment:
Can you rename this to: `_waterfallBannerAd`?

The same applies to the ad formats.

Reply:
ok,I have modified.